config/routes.rb

```diff
@@ -6,11 +6,15 @@
     get "about",   to: "static_pages#about"
     get "help",    to: "static_pages#help"
     get "contact", to: "static_pages#contact"
-    
+
     # Root page
     root "static_pages#home"
 
+    get "/signup", to: "users#new"
+    post "/signup", to: "users#create"
+
     # Resourceful routes
     resources :microposts, only: %i(index)
+    resources :users, only: %i(new create show)
```

Comment:
```suggestion
    resources :users, only: %i(show)
```
Thừa router, new, create mình defind trên sigup rồi